### DISCUSSION TITLE
Can I get VertexBuffer after a gltf model have been draw ?

### DISCUSSION THREAD

**Discussion by fanbinnet** [google/filament#5093] [Q&A]
I need get the vertex position of an model. This model have an animation ,so I can't use Obj format.
If there are some method after the model been draw, I can get the position use the RenderManager ?
Or if can I get the vertex position for per fraction of the animation in an gltf model.

**Comment by pixelflinger** [comment]
No, there is not way to do that.
You will have to transform the vertices yourself.
